docs/guides/advanced.rst

```diff
@@ -151,3 +151,59 @@ differently. Put something like this in your configuration file::
 
 Used together, flexible attributes and path format conditions let you sort
 your music by any criteria you can imagine.
+
+
+Automatically add new music to your library
+-------------------------------------------
+
+As a command-line tool, beets is perfect for automated operation via a cron job
+or the like. To use it this way, you might want to use these options in your
+`config file`_::
+
+    [beets]
+    import_incremental: yes
+    import_quiet: yes
+    import_log: /path/to/log.txt
```

Comment:
This is the old pre-YAML configuration format. The new one would be:

```yaml
import:
    incremental: yes
    quiet: yes
    log: /path/to/log.txt
```

Reply:
I didn't even notice! I guess I was too focused on the change in markup languages and didn't read the actual content carefully. Thanks for checking